app/controllers/mobile.js

```diff
@@ -73,4 +64,17 @@ export default class MobileController extends Controller {
       );
     }
   }
+
+  @action authorizeDeviceAccess() {
+    this.updateQRAuthStatus(
+      AUTH_STATUS.AUTHORIZED,
+      MOBILE_LOGIN_SUCCESS_MESSAGE,
+    );
+    this.showConfirmModal = false;
+  }
+
+  @action rejectDeviceAccess() {
+    this.updateQRAuthStatus(AUTH_STATUS.REJECTED, REQUEST_CANCEL_MESSAGE);
+    this.showConfirmModal = false;
+  }
```

Comment:
can't they be a single function?
```suggestion
 @action handleDeviceAccess(action) {
  if(action === 'authorise'){
   this.updateQRAuthStatus( AUTH_STATUS.AUTHORIZED, MOBILE_LOGIN_SUCCESS_MESSAGE );
  }else{
  this.updateQRAuthStatus(AUTH_STATUS.REJECTED, REQUEST_CANCEL_MESSAGE);
  }
  this.showConfirmModal = false;
}
```